Makefile

```diff
@@ -1253,6 +1253,13 @@ tools/.mdlintstamp: $(LINT_MD_FILES)
 # Lints the markdown documents maintained by us in the codebase.
 lint-md: lint-js-doc | tools/.mdlintstamp
 
+format-lint-md = tools/lint-md/lint-md.mjs --format $(LINT_MD_FILES)
+.PHONY: format-md
+# Formats the markdown documents maintained by us in the codebase.
+format-md:
+	@$(call available-node,$(run-lint-md))
```

Comment:
shouldn't it be `format-lint-md` instead of `run-lint-md` ?

Reply:
Oops! Fixed and tested. Thanks.